### PR TITLE
add mdbootstrap@4.1.1 w/ npm auto-update

### LINKED ISSUE #8906: [Request] Add MDBootstrap
**Library name: MDBoostrap
**Git repository url: https://github.com/mdbootstrap/bootstrap-material-design
**npm package url(optional): mdbootstrap
**License(s): MIT
**Official homepage: http://mdbootstrap.com
**Wanna say something? Leave message here: Material Design for Bootstrap (MDB) is a powerful Material Design UI KIT for most popular HTML, CSS, and JS framework - Bootstrap.

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8906 
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mdbootstrap/bootstrap-material-design
 * Official website (optional, not the repository): https://mdbootstrap.com/material-design-for-bootstrap/
 * NPM package url (optional): https://www.npmjs.com/package/mdbootstrap
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 96
   - Count of watchers: 13
   - Count of forks: 24
 * NPM download stats (optional):
   - Downloads in the last day: 22
   - Downloads in the last week: 124
   - Downloads in the last month: 124

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8906, cc @mattonit

### CODE REVIEW

**Comment by x09326** [comment]
@kennynaoh 
I'm not sure whether the files should be added.
Do you confirm that with the author?

**Comment by mattonit** [comment]
@x09326 
files that should be added:
css/mdb.css
css/mdb.min.css
js/mdb.js
js/mdb.min.js

**Comment by PeterDaveHello** [comment]
@mattonit Thanks a lot!

**Comment by PeterDaveHello** [comment]
LGTM

**Comment by kennynaoh** [comment]
LGTM

**Comment by x09326** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
LGTM

**Comment by kennynaoh** [comment]
Thanks @mattonit ,
I have added libs you mentioned.
And I have also added `font/roboto/*`, `img/overlays/*`, right?
Please help me confirm that.
Thanks!

**Comment by mattonit** [comment]
@kennynaoh, yes `font/roboto/*` and `img/overlays/*` should be added too

thanks!

**Comment by kennynaoh** [comment]
@x09326 
So Please help me take a look again.
Thanks!

**Comment by x09326** [comment]
@kennynaoh 
Please rebase it.

**Comment by kennynaoh** [comment]
@x09326 
I have rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
LGTM

File: ajax/libs/mdbootstrap/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "author": "MDBootstrap",
+  "license": "MIT",
+  "homepage": "https://mdbootstrap.com/material-design-for-bootstrap/",
+  "npmName": "mdbootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(js|css)/mdb*",
```
Comment (PeterDaveHello) [inline ajax/libs/mdbootstrap/package.json]: What about `"+(js|css)/**/mdb*"`?

File: ajax/libs/mdbootstrap/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "font/roboto/*.+(eot|ttf|woff|woff2)",
```
Comment (PeterDaveHello) [inline ajax/libs/mdbootstrap/package.json]: I prefer to replace `roboto` with `**`

File: ajax/libs/mdbootstrap/package.json (hunk 1 above)
Comment (kennynaoh) [inline ajax/libs/mdbootstrap/package.json]: Thanks @PeterDaveHello ,
I want to ask something.
How did you determine that we need `**` under `css|js` folder in this case?
Thanks!

**Comment by kennynaoh** [comment]
@x09326 
I have updated the `npmFileMap`.
Please help me review it again.
Thanks!

File: ajax/libs/mdbootstrap/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/mdbootstrap/package.json]: We don't "need" it, it's for better compatibility, for the future relases.

**Comment by PeterDaveHello** [comment]
Non-trival auto-update also needs multi reviewers.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
LGTM

**Pull request merged**